index.html

```diff
@@ -38,7 +38,133 @@ <h2>LET'S CONNECT</h2>
+            <li class="tag">javascript</li>
+        </ul>
+    <button>see project</button>
+</div>
+</section>
+
+<section class="hero">
+<div class="card">
+    <img src="img/Snapshoot Portfolio(1).png" alt="">
+    <h1>Multi-Post
+        Stories</h1>
+    <ul class="profesional">
+        <li class="item-can">CANOPY </li>
+        <li class="item-tag">&#9679; Back End Dev &#9679;</li>
+        <li class="item-tag">2015</li>
+    </ul>
+    <p>A daily selection of 
+        privately personalized reads; no accounts
+         or sign-ups required.</p>
+        <ul class="category">
+            <li class="tag">html</li>
+            <li class="tag">css</li>
+            <li class="tag">javascript</li>
+        </ul>
+    <button>see project</button>
+</div>
+</section>
+
+<section class="hero">
+<div class="card">
+    <img src="img/Snapshoot Portfolio(2).png" alt="">
+    <h1>Tonic</h1>
+    <ul class="profesional">
+        <li class="item-can">CANOPY </li>
+        <li class="item-tag">&#9679; Back End Dev &#9679;</li>
+        <li class="item-tag">2015</li>
+    </ul>
+    <p>A daily selection of 
+        privately personalized reads; no accounts
+         or sign-ups required.</p>
+        <ul class="category">
+            <li class="tag">html</li>
+            <li class="tag">css</li>
+            <li class="tag">javascript</li>
+        </ul>
+    <button>see project</button>
+</div>
+</section>
+
+<section class="hero">
+<div class="card">
+    <img src="img/Snapshoot Portfolio(3).png" alt="">
+    <h1>Multi-Post
+        Stories</h1>
+    <ul class="profesional">
+        <li class="item-can">CANOPY </li>
+        <li class="item-tag">&#9679; Back End Dev &#9679;</li>
+        <li class="item-tag">2015</li>
+    </ul>
+    <p>A daily selection of 
+        privately personalized reads; no accounts
+         or sign-ups required.</p>
+        <ul class="category">
+            <li class="tag">html</li>
+            <li class="tag">css</li>
+            <li class="tag">javascript</li>
+        </ul>
+    <button>see project</button>
+</div>
+</section>
+<section class="hero">
+<div class="about-me">
+    <h1 class="about-myself">About<br>
+        Myself</h1>
+    <p>Hello I’m a software developer! I can help you build a product ,
+         feature or website Look through some of my work and experience!
+         If you like what you see and have a project you need coded,
+          don’t hestiate to contact me..</p>
+          <h2>LET’S CONNECT</h2>
+          <div class="social-media">
+            <ul>
+                <li><a href="#"><i class=" fa fa-twitter" ></i></a></li>
+                <li><a href="#"><i class="fa fa-linkedin"  ></i></a></li>
+                <li><a href="#"><i class="fa brand fa fa-medium" ></i></a></li>
+                <li><a href="#"><i class="fa fa-github" aria-hidden="true"></i></a></li>
+                <li><a href="#"><i class="fa brand fa fa-angellist" ></i></a></li>
```

Comment:
The size of the `Languages` is a little bit different: i.e.
```
font-size: 20px;
font-weight: 500;
```
Kindly update them.

| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/61361037/221023057-a79d3e9c-bc38-44d3-9a8d-db7b96658a8c.png)| ![image](https://user-images.githubusercontent.com/61361037/221022988-bdcd65c5-0b43-4a92-83f5-83277f980c5e.png)|